Radzen.Blazor/QueryableExtension.cs

```diff
@@ -292,6 +292,184 @@ public static string ToFilterString<T>(this IEnumerable<RadzenDataGridColumn<T>>
             return "";
         }
 
+        /// <summary>
+        /// Converts a RadzenDataFilter to a Linq-compatibly filter string
+        /// </summary>
+        /// <typeparam name="T">The type that is being filtered</typeparam>
+        /// <param name="filter">The RadzenDataFilter component</param>
+        /// <returns>A Linq-compatible filter string</returns>
+        public static string ToLinqFilterString<T>(this RadzenDataFilter<T> filter)
```

Comment:
Maybe the name should be similar to the other method for DataGrid columns - ToFilterString()

Reply:
Done.